ThingManagerOSGiTest unstable

This test failed in: https://ci.openhab.org/job/PR-openHAB-Core/3758

```
TEST org.openhab.core.thing.internal.ThingManagerOSGiTest#thingManagerCallsChildHandlerInitializedAndChildHandlerDisposedOnBridgeHandlerCorrectly() <<< ERROR: 
Expected: is <org.openhab.core.thing.internal.ThingImpl@c7a74f8e>
     but: was null
java.lang.AssertionError: 
Expected: is <org.openhab.core.thing.internal.ThingImpl@c7a74f8e>
     but: was null
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.openhab.core.thing.internal.ThingManagerOSGiTest.thingManagerCallsChildHandlerInitializedAndChildHandlerDisposedOnBridgeHandlerCorrectly(ThingManagerOSGiTest.java:1824)
```

--- a/itests/org.openhab.core.thing.tests/src/main/java/org/openhab/core/thing/internal/ThingManagerOSGiTest.java
+++ b/itests/org.openhab.core.thing.tests/src/main/java/org/openhab/core/thing/internal/ThingManagerOSGiTest.java
@@ -1806,9 +1806,9 @@ public class ThingManagerOSGiTest extends JavaOSGiTest {
         doAnswer(new Answer<Void>() {
             @Override
             public @Nullable Void answer(InvocationOnMock invocation) throws Throwable {
-                bridgeState.childHandlerInitializedCalled = true;
                 bridgeState.initializedHandler = (ThingHandler) invocation.getArgument(0);
                 bridgeState.initializedThing = (Thing) invocation.getArgument(1);
+                bridgeState.childHandlerInitializedCalled = true;
                 return null;
             }
         }).when(bridgeHandler).childHandlerInitialized(any(ThingHandler.class), any(Thing.class));
@@ -1816,9 +1816,9 @@ public class ThingManagerOSGiTest extends JavaOSGiTest {
         doAnswer(new Answer<Void>() {
             @Override
             public @Nullable Void answer(InvocationOnMock invocation) throws Throwable {
-                bridgeState.childHandlerDisposedCalled = true;
                 bridgeState.disposedHandler = (ThingHandler) invocation.getArgument(0);
                 bridgeState.disposedThing = (Thing) invocation.getArgument(1);
+                bridgeState.childHandlerDisposedCalled = true;
                 return null;
             }
         }).when(bridgeHandler).childHandlerDisposed(any(ThingHandler.class), any(Thing.class));

